src/gatling/resources/logback.xml

```diff
@@ -10,7 +10,7 @@
 
     <!-- uncomment and set to DEBUG to log all failing HTTP requests -->
     <!-- uncomment and set to TRACE to log all HTTP requests -->
-    <logger name="io.gatling.http.engine.response" level="DEBUG" />
+    <logger name="io.gatling.http.engine.response" level="DEBUGgit " />
```

Comment:
I think this is a typo mistake

Reply:
Not sure where the 'it' came from but it's now just DEBUG